src/components/tables/examples/TableExamples.tsx

```diff
@@ -94,8 +94,8 @@ const buildNewTableStateManually = (data: any, currentState: ITableState, tableC
 const manualModeThunk = (tableOwnProps: ITableOwnProps, shouldResetPage: boolean, tableCompositeState: ITableCompositeState): IThunkAction => {
   return (dispatch: IDispatch, getState: () => { [globalStateProp: string]: any; tables: ITablesState; }) => {
     const currentTableState = getState().tables[tableOwnProps.id];
-    dispatchPreTableStateModification(tableOwnProps, dispatch);
-    $.get('https://raw.githubusercontent.com/toddmotto/public-apis/master/json/entries.json')
+    dispatchPreTableStateModification(tableOwnProps.id, dispatch);
+    $.get('https://jsonplaceholder.typicode.com/comments')
```

Comment:
Result looks like this
![screen shot 2018-02-12 at 2 58 27 pm](https://user-images.githubusercontent.com/4923043/36116717-3b93c6e8-1005-11e8-83a7-b051c8b90b73.png)

Reply:
thanks for that